app/(public)/components/paragraph/sul-contact-card.tsx

```diff
@@ -0,0 +1,153 @@
+"use client";
+
+import Link from "next/link";
+import Image from "next/image";
+import {ClockIcon, EnvelopeIcon, MapPinIcon, PhoneIcon } from "@heroicons/react/24/outline";
+import useLibraryHours from "@/lib/hooks/useLibraryHours";
+import Conditional from "../utils/conditional";
+import {useResizeDetector} from "react-resize-detector";
+import { ContactCardParagraph } from "@/lib/drupal/drupal";
+
+interface ContactCardProps {
+  paragraph: ContactCardParagraph
+	siblingCount?: number
```

Comment:
I think there seems to be some issue with the tab/space places. I'm not sure what could be the issue here. It looks off through this whole file.

Reply:
Thanks, how about now? I was having weird issues with my editor switching between default spacing sizes, so I'm guessing that was related.